drivers/lora/sx1276.c

```diff
@@ -365,7 +365,10 @@ void SX1276SetRfTxPower(int8_t power)
 
 static int sx1276_config_valid(const struct lora_modem_config *config)
 {
-	if (config->frequency == 0)
+	if (config->frequency == 0 ||
+	    config->bandwidth < BW_MIN || config->bandwidth > BW_MAX ||
```

Comment:
I'd prefer to use the values directly instead of an alias.

Reply:
You mean just add the magic value here? I’d like to avoid that since these constants can be reused in the shell to give a more helpful error message.